### DISCUSSION TITLE
openpgp.min.js file not in main repository

### DISCUSSION THREAD

**Discussion by Robzakhour** [openpgpjs/openpgpjs#1685] [Q&A]
Trying to find the openpgp.min.js file for the latest version of openpgp. 
When using an old version I found on cloudfare (4.10.9), I keep getting "t.Set is not a constructor"

**Comment by larabr** [comment]
Hi, we no longer commit the dist files to github, we only release them to npm.
See the README for one option to retrieve the files if needed: https://github.com/openpgpjs/openpgpjs#browser-plain-files .
